src/libraries/System.Private.Xml/src/System/Xml/Schema/BaseValidator.cs

```diff
@@ -175,23 +175,23 @@ public void ValidateText()
             {
                 if (context.IsNill)
                 {
-                    SendValidationEvent(SR.Sch_ContentInNill, XmlSchemaValidator.QNameString(context.LocalName, context.Namespace));
+                    SendValidationEvent(SR.Sch_ContentInNill, XmlSchemaValidator.QNameString(context.LocalName!, context.Namespace!));
```

Comment:
I think we should update the implementation first to use constructor instead of setting fields.. I'll leave this as is for now and create a global issue with suggestions how to improve

Reply:
actually reading the issue right now before filing it, I honestly don't think it's worth fixing comparing to other XML issues we have so will refrain from filing it.